src/main/java/frc/robot/Constants.java

```diff
@@ -1,32 +1,58 @@
-// Copyright 2021-2024 FRC 6328
-// http://github.com/Mechanical-Advantage
-//
-// This program is free software; you can redistribute it and/or
-// modify it under the terms of the GNU General Public License
-// version 3 as published by the Free Software Foundation or
-// available in the root directory of this project.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU General Public License for more details.
-
 package frc.robot;
 
+import static edu.wpi.first.units.Units.Degrees;
 import static edu.wpi.first.units.Units.Inches;
+import static edu.wpi.first.units.Units.KilogramSquareMeters;
 import static edu.wpi.first.units.Units.Meters;
+import static edu.wpi.first.units.Units.Pounds;
 
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.units.measure.Angle;
 import edu.wpi.first.units.measure.Distance;
+import edu.wpi.first.units.measure.Mass;
+import edu.wpi.first.units.measure.MomentOfInertia;
 import edu.wpi.first.wpilibj.RobotBase;
 
-/**
- * This class defines the runtime mode used by AdvantageKit. The mode is always "real" when running
- * on a roboRIO. Change the value of "simMode" to switch between "sim" (physics sim) and "replay"
- * (log replay from a file).
- */
 public final class Constants {
+  public static class OperatorConstants {
+    public static final int kDriverControllerPort = 0;
+    public static final int kOperatorControllerPort = 1;
+  }
+
+  public static class MotorIDConstants {
+    public static final int kCLimberMotorLeader = 0;
+    public static final int kCLimberMotorFollower = 0;
+  }
+
+  public static class ClimberConstants {
+    public static final double kClimberP0 = 1;
+    public static final double kClimberI0 = 0;
+    public static final double kClimberD0 = 0;
+    public static final double kClimberkG0 = 0;
+    public static final double kClimberkV0 = 2.5;
+
+    public static final double kClimberP1 = 2;
+    public static final double kClimberI1 = 0;
+    public static final double kClimberD1 = 0;
+    public static final double kClimberkG1 = 0.26;
+    public static final double kClimberkV1 = 2.5;
+
+    public static final double KGearRatio = 126;
+    public static final Angle kClimberExtended = Degrees.of(190);
+    public static final Angle kClimberCage = Degrees.of(170);
+    public static final Angle kClimberRetracted = Degrees.of(0);
+
+    public static class ClimberSimConstants {
```

Comment:
This does not need to be two different classes. It's cleaner to just have them in the same class, but have a comment above all the constants that are for sim.